src/compass/defaults/s1_cslc_geo.yaml

```diff
@@ -58,6 +58,11 @@ runconfig:
               low_band_bandwidth: 10000000
               # High-band bandwidth in Hz
               high_band_bandwidth: 10000000
+          correction_luts:
+              # LUT spacing in x/slant range direction in meters
+              x_spacing: 500
+              # LUT spacing in y/azimuth direction in seconds
+              y_spacing: 50
```

Comment:
Not sure if 50 **seconds** of spacing in azimuth direction makes sense. Maybe spacing in meters?

Reply:
Your comment made me notice an inconsistency. So far, the only two corrections that we have (e.g. bistatic delay and geometrical + steering Doppler), they do require the spacings along azimuth and slant range expressed in terms of number of pixels. See https://github.com/opera-adt/s1-reader/blob/4000bc7a31d79e740212ab31495ca28ad0e2d426/src/s1reader/s1_burst_slc.py#L424 . To me, it makes sense to have it expressed in meters/seconds. How do we solve this? Shall we solve this in the s1-reader and change the steps parameters to support meters and seconds or we do the conversion internally? What do you think @LiangJYu ?

Reply:
Will these correction LUTs only be used by the core ISCE3 modules? If so then I think we should alter `burst.bistatic_delay` and `burst.geometrical_and_steering_doppler` to return LUT2d that support meters and seconds.

If there are other processes that expect these LUT2ds in pixels, we leave s1_reader untouched and change `compass.utils.lut.compute_geocoding_correction_luts` to support meters and seconds.

What do you all think?

Reply:
Well, the primary user would be the `geocodeSlc` core module. However, providing a spacing in seconds in the runconfig/schema seems a bit counter intuitive to me. I am not sure which is the right approach. Shall we go ahead and modify the two corrections in the `s1-reader` to support meters and seconds? Are those two corrections used somewhere else (e.g. RTC)?